### PR TITLE
restore errantly dropped tests.

### PR DESCRIPTION
**Pull request by higs4281**
Restore two cucumber tests that were errantly deleted.

### CODE REVIEW

File: test/browser_tests/cucumber/features/suites/default/pagination.feature [hunk 1]
```diff
@@ -10,3 +10,13 @@ Feature: Pagination
   Scenario: Navigate to the next page
     When I click on the next button
     Then the page url should contain "page=2"
+
+  Scenario: Navigate to the previous page
+    When I click on the next button
+    And I click on the previous button again
+    Then the page url should contain "page=1"
+
+   Scenario: Navigate to nth Page
```
Comment (anselmbradford) [inline test/browser_tests/cucumber/features/suites/default/pagination.feature]: This one is indented an extra space.

Comment (higs4281) [reply test/browser_tests/cucumber/features/suites/default/pagination.feature]: thanks @anselmbradford 
How many ways can I screw this up?

**Comment by willbarton** [comment]
It looks like it's [still failing on the pagination test](https://travis-ci.org/cfpb/cfgov-refresh/jobs/533341749#L1104)?

**Comment by higs4281** [comment]
Yes, this doesn't fix anything, just restores two tests I didn't mean to delete.
I don't want to move the baseline for troubleshooting.

**Pull request merged**